kuwala/core/canvas/src/api/DataSourceApi.js

```diff
@@ -7,3 +7,17 @@ import {
 export function getDataSource() {
     return baseAxios.get(DATA_SOURCE)
 }
+
+export function testConnections(data) {
```

Comment:
Since it's only testing the connection for one data source it should be called `testConnection`.